pages/landing/hipaa/template/index.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Achieve HIPAA compliant infrastructure, instantly
+excerpt: Go live with a customized, HIPAA-compliant cloud infrastructure and sample application in about a day.
+permalink: /hipaa-cloud-compliance/
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Join the Waitlist
+  logos:
+    - /assets/img/logos-technologies/partner_logo_aws.png
+    - /assets/img/logos-compliance/cis.png
+    - /assets/img/logos-compliance/hipaa.png
+
+how_it_works:
+  - heading: Deploy on HIPAA-compliant infrastructure
+    image: /assets/img/infrastructure-cube-icons/infrastructure-hipaa.png
```

Comment:
Updated URL: https://deploy-preview-453--keen-clarke-470db9.netlify.app/migrate-from-heroku-to-aws/#step-1-connect-your-git-repo

Reply:
Some thoughts on things it could make sense to demonstrate: 
* The current AWS sample app code middleware - but with a clearly visible comment block that says "write your code here" where the user is typing in their own Javascript code
* A UI mock that shows a Business Associate Addendum being executed with an easy electronic signature and one button click
* A UI mock of a Github release page showing one of the Gruntwork module's being updated with compliance changes + API improvements - perhaps juxtaposed against a peel-back view of the user entering the release's tag into their own module's source URL.

Reply:
Eben and Pete and I discussed new graphics which are forthcoming from Eben - so I'm going to resolve this convo for now.